bin/cli.js

```diff
@@ -38,14 +40,16 @@ if (argv.h) {
   console.log('')
   console.log('Options:')
   console.log('')
-  console.log('  -d, --diff             output diff against original file')
-  console.log('  -l, --list             format list of space seperated files in place')
-  console.log('  -R, --recursive        format files recursively')
-  console.log('  -c, --config           path to a specific configuration file (JSON, YAML, or CommonJS)')
-  console.log('  -b, --config-basedir   path to the directory that relative paths defining "extends"')
-  console.log('  -v, --version          output the version number')
-  console.log('  -h, --help             output usage information')
-  process.exit()
+  console.log('  -d, --diff             Output diff against original file')
+  console.log('  -l, --list             Format list of space seperated files in place')
+  console.log('  -R, --recursive        Format files recursively')
+  console.log('  -c, --config           Path to a specific configuration file (JSON, YAML, or CommonJS)')
+  console.log('  -b, --config-basedir   Path to the directory that relative paths defining "extends"')
+  console.log('  -v, --version          Output the version number')
+  console.log('  -h, --help             Output usage information')
+  console.log('  -i, --ignore-path      Path to a file containing patterns that describe files to ignore.')
+  console.log('  -id --ignore-disables  Ignore disables')
+  console.log('  --stdin-filename       A filename to assign stdin input.')
```

Comment:
[sublime-stylefmt](https://github.com/dmnsgn/sublime-stylefmt) Have a [stylefmt caller](https://github.com/dmnsgn/sublime-stylefmt/blob/master/stylefmt.js).

I think `sublime-stylefmt` should just call `cli.js` of `stylefmt`, not creat other caller, so we need this API

`sublime-stylefmt` should just call `stylefmt --stdin-filename /path/to/a/change/unsaved/file`

If user of SublimeText call stylefmt before save file, we need pass file path and code, Not a single one can be omitted